.npmignore

```diff
@@ -1,8 +1,14 @@
-.DS_Store
 *.log
-examples
-test
-coverage
-_book
-book.json
-docs
+/.*/
```

Comment:
I am strictly disagree. This is developers own folder and it is not in project. Any application can create any folders in project and that doesn't mean we need to ignore all possible folder names. You should add `.idea` to your global gitignore